src/Explorer/Explorer.tsx

```diff
@@ -85,6 +86,12 @@ export default class Explorer {
   // Notebooks
   public notebookManager?: NotebookManager;
 
+  public conversationToken: ko.Observable<string>;
```

Comment:
I think we were trying to migrate DE away from knockout. Any reason why we would want these properties to be ko?